client/src/pages/LyricsView/LyricsDisplay.tsx

```diff
@@ -7,32 +7,58 @@ import {
   InputAdornment,
   Link,
   makeStyles,
+  Toolbar,
   TextField,
   Typography
 } from "@material-ui/core";
 import SearchIcon from "@material-ui/icons/Search";
 import CloseIcon from "@material-ui/icons/Close";
 
+import SyncEnabledIcon from "@material-ui/icons/Sync";
+import SyncDisabledIcon from "@material-ui/icons/SyncDisabled";
+
 import "./LyricsDisplay.css";
+import { Sync } from "@material-ui/icons";
+
+// adjusting for latency to highlight lyrics due to the time it takes to render the components on screen
+const LATENCY_ADJUSTMENT_MAGIC_VALUE: number = 135;
+
+interface ILRCContent {
+  content: string;
+  timestamp: number;
+}
```

Comment:
You have this in client\src\types\trackLyrics.ts - can reference that here and drop this